sections/04_study.md

```diff
@@ -389,7 +390,7 @@ align sequences and predict bulges, mismatches, and wobble base pairing without
 requiring the user to input secondary structure predictions or thermodynamic
 calculations.
 
-Further incremental advances in neural-network approaches for miRNA and target
+Further incremental advances in neural network approaches for miRNA and target
```

Comment:
Yes. It reads a little strange in that *we're* talking about DL and there's a statement about improvements in NN. There is a quandary here, probably can solve by more clearly labelling as early work.